### PR TITLE
utils_iptables: Moving setup_or_cleanup_iptables_rules()

### PR DESCRIPTION
**Pull request by balamuruhans**
API setup_or_cleanup_iptables_rules() to configure and cleanup
iptable rules for the tests to handle iptables related configurations.

Signed-off-by: Balamuruhan S <bala24@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by balamuruhans** [comment]
@will-Do, have moved the method to test_setup.py and cyclic import issue is not observed.
please help me in merging the PR.

**Tested:**
```
2017-04-19 05:05:19,446 remote           L0362 DEBUG| Attempting to log into 9.40.192.88:22 using ssh (timeout 240s)
2017-04-19 05:05:26,858 client           L1049 DEBUG| Sending command: iptables -S
2017-04-19 05:05:27,059 client           L1049 DEBUG| Sending command: echo $?
2017-04-19 05:05:27,210 test_setup       L2121 DEBUG| Rule: INPUT -p tcp -m tcp --dport 49152:49216 -j ACCEPT exist in iptables
2017-04-19 05:05:27,372 remote           L0362 DEBUG| Attempting to log into 9.40.192.88:22 using ssh (timeout 240s)
2017-04-19 05:05:34,829 client           L1049 DEBUG| Sending command: grep cpu /proc/cpuinfo | awk '{print $3}' | head -n 1
2017-04-19 05:05:34,988 client           L1049 DEBUG| Sending command: echo $?
2017-04-19 05:05:35,298 remote           L0362 DEBUG| Attempting to log into 9.40.192.88:22 using ssh (timeout 240s)
2017-04-19 05:05:42,692 client           L1049 DEBUG| Sending command: ppc64_cpu --smt
2017-04-19 05:05:42,854 client           L1049 DEBUG| Sending command: echo $?
```

**Comment by balamuruhans** [comment]
@will-Do please help in merging this PR

**Comment by will-Do** [comment]
Thanks @balamuruhans 
Moving this function is fine for usage, but for a better code structure and maintain, that could be nice if you could create a new module (such as merge it to virttest/utils_net.py or a new file: utils_iptables.py)to handle all iptables related works, so we can improve it time by time to meet more requirements. What do you think?

**Comment by balamuruhans** [comment]
@will-Do I Agree, I will create new module in utils_iptables.py

**Comment by balamuruhans** [comment]
**Log**
```
2017-05-02 08:10:56,562 remote           L0362 DEBUG| Attempting to log into 9.40.192.88:22 using ssh (timeout 240s)
2017-05-02 08:11:03,930 client           L1049 DEBUG| Sending command: iptables -S
2017-05-02 08:11:04,134 client           L1049 DEBUG| Sending command: echo $?
2017-05-02 08:11:04,286 utils_iptables   L0061 DEBUG| Adding rule: INPUT -p tcp -m tcp --dport 49152:49216 -j ACCEPT
2017-05-02 08:11:04,286 client           L1049 DEBUG| Sending command: iptables -I INPUT -p tcp -m tcp --dport 49152:49216 -j ACCEPT
2017-05-02 08:11:04,441 client           L1049 DEBUG| Sending command: echo $?
2017-05-02 08:11:04,593 utils_iptables   L0073 DEBUG| iptable command success iptables -I INPUT -p tcp -m tcp --dport 49152:49216 -j ACCEPT
2017-05-02 08:11:04,752 remote           L0362 DEBUG| Attempting to log into 9.40.192.88:22 using ssh (timeout 240s)
```

**Comment by balamuruhans** [comment]
@will-Do, added the API `setup_or_cleanup_iptables_rules()` in utils_iptables.py

File: virttest/utils_iptables.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+"""
+import logging
+
+from avocado.utils import process
+from avocado.core import exceptions
+
+from . import remote
+
+
+class Iptables():
```
Comment (xutian) [inline virttest/utils_iptables.py]: class Iptables(object):

File: virttest/utils_iptables.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    """
+    class to handle all iptables configurations related methods
+    """
+
+    def setup_or_cleanup_iptables_rules(self, rules, params=None,
```
Comment (xutian) [inline virttest/utils_iptables.py]: looks like a @classmethod

**Comment by balamuruhans** [comment]
@xutian, have made changes as per your suggestion.

**Comment by xutian** [comment]
looks OK to me.

**Comment by balamuruhans** [comment]
@xutian Thank you, let us merge this

**Pull request merged**